### DISCUSSION TITLE
How to adjust learning rate while trainning?

### DISCUSSION THREAD

**Discussion by NeroBlackstone** [LuxDL/Lux.jl#777] [Q&A] [answered]
Is there any [learning rate schedulers](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate) alternative in Lux.jl?

**Comment by avik-pal** [comment, marked as answer]
https://github.com/FluxML/ParameterSchedulers.jl

Example Usage: https://github.com/LuxDL/Lux.jl/blob/2ec21e23208c7e29ad17585088f0a7c01d3d5a6a/examples/ImageNet/main.jl#L281
